graph/src/data/subgraph/schema.rs

```diff
@@ -58,17 +58,48 @@ pub trait TypedEntity {
 
 #[derive(Debug)]
 pub struct SubgraphEntity {
-    name: SubgraphName,
-    current_version_id: Option<String>,
-    pending_version_id: Option<String>,
-    created_at: u64,
+    pub name: SubgraphName,
+    pub current_version_id: Option<String>,
+    pub pending_version_id: Option<String>,
+    pub created_at: u64,
 }
 
 impl TypedEntity for SubgraphEntity {
     const TYPENAME: &'static str = "Subgraph";
     type IdType = String;
 }
 
+impl TryFromEntity for SubgraphEntity {
+    const ENTITY_TYPE: &'static str = "Subgraph";
```

Comment:
How about merging `TryFromEntity` with `TypedEntity` to avoid the duplication here? Then `get_typed<T>` would be generic over `TypedEntity`, which makes sense

Reply:
Good idea, will do that.